workspaces/ui/src/services/diff/LocalCliDiffService.ts

```diff
@@ -46,7 +46,7 @@ export class LocalCliDiffService implements IDiffService {
 
   async listUnrecognizedUrls(): Promise<IListUnrecognizedUrlsResponse> {
     const url = `${this.baseUrl}/undocumented-urls`;
-    const json = (await JsonHttpClient.getJson(url)).urls;
+    const json = await JsonHttpClient.getJson(url);
```

Comment:
I suspect this might result in different behaviors when using the example data vs local cli, we'll have to make sure we don't break those.

Reply:
Ahh, I would have expected this `LocalCliDiffService` to only talk to the `cli-server`, and the `ExampleDiffService` to take care of the example data. If that's not the case, maybe we'll want to refactor things to be like that, as that would be pretty confusing 😅.

Reply:
Or we can get the cli-server to adhere to the local API and figure out the wrapping of the streamed items!